diff -Nru benchmark-1.4.1/src/sysinfo.cc benchmark-1.4.1.rasm/src/sysinfo.cc
--- benchmark-1.4.1/src/sysinfo.cc	2018-05-25 11:18:58.000000000 +0100
+++ benchmark-1.4.1.rasm/src/sysinfo.cc	2018-09-02 23:59:43.000000000 +0100
@@ -289,10 +289,10 @@
     std::string type;
     int level;
     size_t num_sharing;
-  } Cases[] = {{"hw.l1dcachesize", "Data", 1, CacheCounts[1]},
-               {"hw.l1icachesize", "Instruction", 1, CacheCounts[1]},
-               {"hw.l2cachesize", "Unified", 2, CacheCounts[2]},
-               {"hw.l3cachesize", "Unified", 3, CacheCounts[3]}};
+  } Cases[] = {{"hw.l1dcachesize", "Data", 1, (size_t)CacheCounts[1]},
+               {"hw.l1icachesize", "Instruction", 1, (size_t)CacheCounts[1]},
+               {"hw.l2cachesize", "Unified", 2, (size_t)CacheCounts[2]},
+               {"hw.l3cachesize", "Unified", 3, (size_t)CacheCounts[3]}};
   for (auto& C : Cases) {
     int val;
     if (!GetSysctl(C.name, &val)) continue;
